### PR TITLE
Adds a DLRM Tensorflow test that runs on a v5p-8

### PR DESCRIPTION
**Pull request by EricLeFort**
# Description

Adds a v5p-8 DLRM test

Also does some other minor housekeeping changes

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**List links for your tests (use go/shortn-gen for any internal link):**
Nightly tests are failing in the same way as they are currently: http://shortn/_NvBaOUspmv http://shortn/_XR5kKdSdKn http://shortn/_fVMe30WNnY
Keras on tf 2.15: http://shortn/_Vgcnd0vVMQ
Keras + Resnet on tf se 2.15: http://shortn/_Nfb9IA09Xs http://shortn/_UsYbrGAtmw
DLRM v5p (second run shows clearing the directory for checkpointing):  http://shortn/_YlWBZUs47B http://shortn/_jn8lFG3GPv

DLRM v5p with the checkpoint path change: http://shortn/_FAJPUle0QF

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [X] I have performed a self-review of my code.
- [X] I have necessary comments in my code, particularly in hard-to-understand areas.
- [X] I have run one-shot tests and provided workload links above if applicable. 
- [X] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

**Review by chandrasekhard2 — APPROVED** [review]

**Review by will-cromar — APPROVED** [review]

File: dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py [hunk 1]
```diff
@@ -289,19 +301,21 @@ def get_tf_dlrm_config(
...
-  benchmark_id = f"{test_name}-v{tpu_version.value}-{tpu_cores}"
-  # Add default_var to pass DAG check
-  # TODO(ranran): replace Variable.get() to XCOM when it applies
-  tpu_name = Variable.get(benchmark_id, default_var=None) if is_pod else "local"
-  env_variable = export_env_variable(is_pod, is_pjrt)
+  model_dir = "/tmp/"
+  if is_v5p:
+    params_override["trainer"]["pipeline_sparse_and_dense_execution"] = "true"
+    tpu_id = Variable.get(benchmark_id, default_var=None)
+    model_dir = gcs_bucket.BASE_OUTPUT_DIR + f"/dlrm/v5p/{benchmark_id}"
```
Comment (RissyRan) [inline dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py:308]: Wondering why model_dir is on gcs for v5p?

Also, could you update it to something like 

```
gcs_bucket.BASE_OUTPUT_DIR + f"{Team.SOLUTIONS_TEAM.value}/dlrm/v5p/{benchmark_id}"
```

We actually added a new feature to use default gcs location. Could you take a quick look and see if that fits as well? One example is [here](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/solutions_team/configs/flax/solutionsteam_flax_latest_supported_config.py#L188): 
* check `gcs_subfolder` to pass in team's info
* check `metric_config.SshEnvVars.GCS_OUTPUT.value` to get this value
* check `use_runtime_generated_gcs_folder` as True

**Review by RissyRan — COMMENTED** [review]
Just have one commnet

File: dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py (hunk 1 above)
Comment (RissyRan) [reply dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py:308]: Thanks for the explanation! Yeah, if it has multi-hosts, using gcs bucket for ckpts makes sense. 

Will adding `gcs_subfolder` and `metric_config.SshEnvVars.GCS_OUTPUT.value` works for you. No need to call `use_runtime_generated_gcs_folder` this is for metric processing.

Comment (EricLeFort) [reply dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py:308]: Yup, that works, will fix!

(I thought the `metric_config` was an instance when I read that first but it's actually the package name.)

**Review by RissyRan — APPROVED** [review]

File: dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py (hunk 1 above)
Comment (RissyRan) [reply dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py]: Discussed offline. We will use default location later as env is not imported if did not indicate `use_runtime_generated_gcs_folder` flag.
